pallets/proposals/src/weights.rs

```diff
@@ -39,6 +40,11 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
             .saturating_add(T::DbWeight::get().reads(2_u64))
             .saturating_add(T::DbWeight::get().writes(2_u64))
     }
+    fn update_project() -> Weight {
```

Comment:
This needs a benchmark, writing static weights is not enough, that reminds me we need to regen weights.

Reply:
Agreed. Will work on this on a new PR